include/alpaka/rand/TinyMT/tinymt32.h

```diff
@@ -1,10 +1,5 @@
 /* Copyright 2020 Axel Huebl, Benjamin Worpitz, Mutsuo Saito, Bernhard Manfred Gruber
- *
- * This file is part of alpaka.
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
```

Comment:
```suggestion
 * SPDX-License-Identifier: BSD-3-Clause
```
I would generally replace the author list above by the the original authors of the file's main content.

Reply:
Good catch. This was added automatically a few years back and is likely an oversight.